bin/client/src/interop/transition.rs

```diff
@@ -105,7 +105,8 @@ where
         beacon,
         l1_provider.clone(),
         l2_provider.clone(),
-    );
+    )
+    .await?;
```

Comment:
Yea that's ok i guess

Reply:
Async instantiators are not my favorite but it's acceptable. At that point a builder pattern is preferred but can be ticketed out after.